Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# 3rd Historical Flavor Crusade

# Changelog:
- Added two major Silithian Twilight's Hammer figures: Highlord Demitrian and Vyral the Vile
- Gordunni ogres should hold the Seething Shore in 605 startdate.
- Added the Falrevere dynasty of Lordaeron nobles-turned-pirates
- Added the Ashmaul clan of ogres
- Added Dr. Paul Lapidis
- Added the Bloodsail Commodores based on RPG lore (Wallace Boltscrew, Lester Zank, Anetta Crank and Quentin Bandor)
- Colton Falrevere is now a duke of Channel Islands under Kul Tiras in 583 startdate, but leaves for Plunder Isle in 600, becoming the leader of Bloodsail Buccaneers.
- Gwen Falrevere begins as a ruler of Silverpine duchy under Lordaeron in 583 and 603 startdates, being a level 3 priest. She also joins the Forsaken in 605 as a courtier after Lordaeronian collapse. This was done to establish the Falrevere historical holdings for earlier startdates.
- Lord Mason is now a vassal of Silverpine and was moved to County of Deathknell, which he holds in 583 and 603 startdates.
- Re-arranged the setup of Plunder Isle. It should be held mostly by natives at the first bookmark (583). Paul Lapidis holds his eponymous island in all startdates. In latter two bookmarks, there are major changes to the political landscape of the region, such as Alex the Australian leaving Un'gol and defecting from Kul Tiras to Bloodsails and then taking control of Northpoint. Ashmaul ogres occupy Un'gol, while the pirate crew is now properly held by Duke Colton Falrevere, while Firallon is relegated to South Island (aka Jaguero Isle) of Stranglethorn.
- Renamed Forlorn_3 and South Isle to Bloodsail and Jaguero, respectively
- Added Fleet Master Seahorn who now appears in the court of Booty Bay's Baron Revilgaz.

# How to test:
- Verify if the culture/religion setups are properly set in every startdate for both the provinces and characters.
- Verify if every new or moved character appears properly in courts.
